### PR TITLE
Adding adopters template

### PR DESCRIPTION
**Pull request by fcarta29**
Adding template for adopters list

### CODE REVIEW

**Review by nimakaviani — APPROVED** [review]
LGTM. thanks for this!

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Who is using CNOE?
```
Comment (nabuskey) [inline ADOPTERS.md:1]: CNOE or idpbuilder?

Comment (nimakaviani) [reply ADOPTERS.md:1]: yeah idpBuilder makes more sense. @fcarta29 can you make the change plz?

**Review by nimakaviani — APPROVED** [review]
LGTM

**Comment by nimakaviani** [comment]
@fcarta29 you also need to sign the DCO :)

**Comment by nabuskey** [comment]
Closing this due to inactivity. Feel free to reopen.

**Pull request closed without merging**